src/daft-catalog/src/error.rs

```diff
@@ -75,7 +75,7 @@ impl Error {
                 .into_iter()
                 .map(Into::into)
                 .collect::<Vec<_>>()
-                .join("."),
+                .join(","),
```

Comment:
```suggestion
                .join(", "),
```

Added whitespace after the comma, thanks for cleaning up this message.